py/README.rst

```diff
@@ -83,30 +83,30 @@ For running on YARN and other supported platforms please see `Running Sparkling
 
       from pysparkling import *
       import h2o
-      hc = H2OContext(sc).start()
+      hc = H2OContext.getOrCreate(sc)
 
 
 Run IPython Notebook with PySparkling
 -------------------------------------
 .. code-block:: bash
 
-    IPYTHON_OPTS="notebook" bin/pysparkling
+    IPYTHON_OPTS="notebook"
```

Comment:
Oh, ok, I'll rather revert that.

Thanks,
-Vlad

On Tue, Aug 30, 2016 at 5:42 PM, Michal Malohlava <notifications@github.com>
wrote:

> In py/README.rst
> <https://github.com/h2oai/sparkling-water/pull/66#discussion_r76906514>:
>
> >
> >
> >  Run IPython Notebook with PySparkling
> >  -------------------------------------
> >  .. code-block:: bash
> >
> > -    IPYTHON_OPTS="notebook" bin/pysparkling
> > +    IPYTHON_OPTS="notebook"
>
> This need to be together on the same line because of bash:
> so it can be:
>
>  IPYTHON_OPTS="notebook" bin/pysparkling
>
> or
>
> export IPYTHON_OPTS="notebook"
> bin/pysparkling
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/h2oai/sparkling-water/pull/66/files/facb9e6ff23734f27c6c540f5d589d743a61fc1b#r76906514>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOuK7YvnMraG3lNEREK0_WfQV-3XUDwks5qlM3-gaJpZM4Jw-AK>
> .
>